.circleci/config.yml

```diff
@@ -123,7 +144,18 @@ jobs:
       - image: circleci/node:10.18
     environment:
       CODE_PATTERN: (<i )|(<i$)
-      OVERALL_REVIEW_COMMENT: Found `<i>` tag in changes. _Please_ wait for a VSP review.
+      OVERALL_REVIEW_COMMENT: >
+        # Icon found
+
+        Icons can be decorative, but sometimes they are used to convey meaning.
```

Comment:
@1Copenut is there something we can link to that elaborates on this? maybe in the design system?

Reply:
@rianfowler I looked through design.va.gov and there's not anything specific to this point (though there should be). What usually happens with the `<i>` icons being flagged, they exist without an `aria-hidden="true"` attribute. JAWS treats them like a tab stop, but doesn't announce them, which is confusing. As long as they have descriptive text nearby, they are decorative and should be hidden.